WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -648,6 +653,49 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
         mediaInserterHelper.cancelUploadOf(media: media)
     }
 
+    func gutenbergDidRequestToSetFeaturedImage(for mediaID: Int32) {
+        // dismiss current controller in order to allow the new alert controller to be presented
+        presentedViewController?.dismiss(animated: false, completion: nil)
+
+        let featuredImageId = post.featuredImage?.mediaID
+        let mediaIdNoFeaturedImageSet = 0
+
+        if featuredImageId != nil && mediaID == featuredImageId as! Int32 {
+            // nothing special to do, trying to set the image that's already set as featured
+            return
+        }
+
+        if mediaID == mediaIdNoFeaturedImageSet {
+            // user tries to clear the featured image setting
+            featuredImageHelper.setFeaturedImage(mediaID: mediaID)
+            return
+        }
+
+        if featuredImageId == nil {
+            // current featured image is not set so, go ahead and set it to the provided one
+            featuredImageHelper.setFeaturedImage(mediaID: mediaID)
+            return
+        }
+
+        // ask the user to confirm changing the featured image since there's already one set
+        showAlertForReplacingFeaturedImage(mediaID: mediaID)
+    }
+
+    func showAlertForReplacingFeaturedImage(mediaID: Int32) {
+        let alertController = UIAlertController(title: NSLocalizedString("Replace?", comment: "Title message on dialog that prompts user to confirm or cancel the replacement of a featured image."),
+                                                message: NSLocalizedString("You already have a featured image set. Do you want to replace it?", comment: "Main message on dialog that prompts user to confirm or cancel the replacement of a featured image."),
+                                                preferredStyle: .actionSheet)
+
+        let replaceAction = UIAlertAction(title: NSLocalizedString("Replace", comment: "Button to confirm the replacement of a featured image."), style: .default) { (action) in
+            self.featuredImageHelper.setFeaturedImage(mediaID: mediaID)
+        }
+
+        alertController.addAction(replaceAction)
+        alertController.addCancelActionWithTitle(NSLocalizedString("Replace", comment: "Button to cancel the replacement of a featured image."))
```

Comment:
```suggestion
        alertController.addCancelActionWithTitle(NSLocalizedString("Cancel", comment: "Button to cancel the replacement of a featured image."))
```

Reply:
Thanks for spotting this! I've gone ahead to apply the suggestion in https://github.com/wordpress-mobile/WordPress-iOS/pull/16426/commits/51c8c13e06e5084ab128e5d8f3885502868c3920.